src/vcpkg/commands.integrate.cpp

```diff
@@ -247,7 +247,17 @@ namespace vcpkg::Commands::Integrate
             std::error_code ec;
             const auto tmp_dir = paths.buildsystems / "tmp";
             fs.create_directory(paths.buildsystems, ec);
```

Comment:
```suggestion
            fs.create_directory(paths.buildsystems, VCPKG_LINE_INFO);
```

That will cause the error to be printed rather than swallowed. Ditto below.